### ISSUE TITLE
Missing function icon inheritance

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1177]
**Reported by beutlich on 25 Jun 2013 15:30 UTC**
The following (protected) functions are not derived from Modelica.Icons.Function:
```
Modelica.Blocks.Continuous.Internal.Filter.Utilities.normalizationFactor.normalizationResidue
Modelica.Blocks.Continuous.Internal.Filter.Utilities.normalizationFactor.findInterval
Modelica.Blocks.Continuous.Internal.Filter.Utilities.normalizationFactor.solveOneNonlinearEquation
Modelica.Blocks.Continuous.Internal.Filter.Utilities.bandPassAlpha.residue
Modelica.Blocks.Continuous.Internal.Filter.Utilities.bandPassAlpha.solveOneNonlinearEquation
Modelica.Blocks.Nonlinear.PadeDelay.padeCoefficients
Modelica.Blocks.Sources.TimeTable.getInterpolationCoefficients
Modelica.Blocks.Sources.BooleanTable.getFirstIndex
Modelica.Blocks.Sources.IntegerTable.getFirstIndex
Modelica.Blocks.Types.ExternalCombiTimeTable.constructor
Modelica.Blocks.Types.ExternalCombiTimeTable.destructor
Modelica.Blocks.Types.ExternalCombiTable1D.constructor
Modelica.Blocks.Types.ExternalCombiTable1D.destructor
Modelica.Blocks.Types.ExternalCombiTable2D.constructor
Modelica.Blocks.Types.ExternalCombiTable2D.destructor
Modelica.Electrical.Analog.Semiconductors.pow
Modelica.Electrical.Analog.Semiconductors.exlin
Modelica.Electrical.Analog.Semiconductors.NPN.pow
Modelica.Electrical.Analog.Semiconductors.PNP.pow
Modelica.Electrical.Digital.Memories.DLATRAM.address
Modelica.Math.Vectors.Utilities.roots
Modelica.Math.Matrices.exp.columnNorm
Modelica.Math.Matrices.integralExp.columnNorm
Modelica.Utilities.Files.list.listFile
Modelica.Utilities.Files.list.sortDirectory
Modelica.Utilities.Files.list.listDirectory
Modelica.Utilities.Files.copy.copyDirectory
Modelica.Utilities.Files.copy.copyDirectoryContents
Modelica.Utilities.Files.remove.removeDirectory
Modelica.Utilities.Files.remove.removeDirectoryContents
Modelica.Utilities.Files.createDirectory.existDirectory
Modelica.Utilities.Files.createDirectory.assertCorrectIndex
Modelica.Utilities.Files.loadResource
Modelica.Utilities.Strings.scanDelimiter.concatenate
Modelica.Utilities.Internal.PartialModelicaServices.ExternalReferences.PartialLoadResource
Modelica.Mechanics.MultiBody.Joints.Internal.RevoluteWithLengthConstraint.selectBranch
Modelica.Mechanics.MultiBody.Joints.Internal.PrismaticWithLengthConstraint.selectBranch
Modelica.Mechanics.Rotational.Components.AngleToTorqueAdaptor.Move_w.position
Modelica.Mechanics.Rotational.Components.AngleToTorqueAdaptor.Move_w.position_der
Modelica.Mechanics.Rotational.Sources.Move.position
Modelica.Mechanics.Rotational.Sources.Move.position_der
Modelica.Mechanics.Rotational.Sources.Move.position_der2
Modelica.Mechanics.Translational.Sources.Move.position
Modelica.Mechanics.Translational.Sources.Move.position_der
Modelica.Mechanics.Translational.Sources.Move.position_der2
```
List is incomplete as I did not check Media and Fluid.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1177

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 25 Jun 2013 16:57 UTC**
Missing function icon extends added in c31f054171d5d49456dd9090376c0b8a8b948a5d

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 25 Jun 2013 16:57 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by beutlich on 26 Jun 2013 07:16 UTC**
... and c615adbe05051b76e4c34b443b20edbb8ed70d1b.

**Comment by modelica-trac-importer** [comment]
**Comment by beutlich on 13 Aug 2013 13:46 UTC**
Should the encapsulated, partial function Modelica.Math.Nonlinear.Interfaces.partialScalarFunction also derive from Modelica.Icons.Function?

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 14 Aug 2013 09:15 UTC**
Fixed in 6398445d26f2358680df0d217736014c4fd51e9c

**Comment by modelica-trac-importer** [comment]
**Modified by otter on 14 Aug 2013 09:16 UTC**
